docgen/content-sources/node/toc.yaml

```diff
@@ -130,6 +130,12 @@ toc:
 - title: "admin.firestore"
   path: /docs/reference/admin/node/admin.firestore
 
+- title: "admin.installations"
+  path: /docs/reference/admin/node/admin.installations
+  section:
+  - title: "Installations"
+    path: /docs/reference/admin/node/admin.installations.Installations-1
+
 - title: "admin.instanceId"
```

Comment:
It's possible to add a "status: deprecated" flag here to create this kind of effect:

https://firebase.google.com/docs/reference/android/com/google/firebase/iid/package-summary?hl=en